letsencrypt/tests/auth_handler_test.py

```diff
@@ -7,14 +7,17 @@
 
 from acme import challenges
 from acme import client as acme_client
+from acme import jose
 from acme import messages
 
 from letsencrypt import achallenges
 from letsencrypt import errors
 from letsencrypt import le_util
 
-from letsencrypt.tests import acme_util
+from letsencrypt.configuration import NamespaceConfig
```

Comment:
I think we can remove the entire import line if ```self.config``` will work if it's simply mocked.